src/main/java/com/worldwarofants/game/arch/console/ConsoleOutputHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.worldwarofants.game.arch.console;
+
+import java.util.concurrent.TimeUnit;
+
+public class ConsoleOutputHandler{
+
+    private static final long SPELL_DELAY = 100;
+
+    //output the provided string in one line, character by character
+    public static void spell(String input) {
```

Comment:
You can make these method comments fancier by turning them into the so called `java docs`. Which is done like this: 
```
/**
* Output the provided string in one line, character by character
*/
```
The main cool thing about this is that you can enable a feature which allows you to mouse over the methods, and see a tool tip with that comment. There's many other really helpful things about java docs, you can read about it online.

Reply:
Updated to increase fancy-ness.